src/main/java/edu/andreasgut/view/OptionsView.java

```diff
@@ -1,44 +1,51 @@
 package edu.andreasgut.view;
 
+import edu.andreasgut.view.internet.MusicSwitchButton;
```

Comment:
Hier importierst Du eine Klasse (MusicSwitchButton) aus einem Paket das nicht existiert! 
Kann sein dass das Paket

    edu.andreasgut.view.internet

bei Dir lokal existiert, aber in diesem Fall hättest Du das noch nicht auf github hochgeladen. Vielleicht ist das dieses Element von dem Du heute berichtet hast, das nicht Teil der Standard-Bibliothek ist.

Reply:
Mit dem Befehl

    git add

kannst Du Dateien und Verzeichnisse zum aktuellen commit hinzufügen (man nennt das dann die "staging area"), und mti einem Commit werden alle Dateien/Verzeichnisse aus der Staging-Area in den Commit übernommen.

Reply:
Hmmmm... Eigentlich hab ich es schon "geaddet". Es erschien jedenfalls grün in IntellJ und sonst sind die Dateien rot. Aber auf github fand ich die Klasse auch nicht. Ich hab es jetzt mal in einen anderen Ordner verschoben und dann einen Commit erstellt, dann hat es die Änderung jedenfalls erkannt und nun finde ich es nach dem push auch auf github. Ist jetzt alles da, was du brauchst?

Reply:
Ich habe diesen Branch bei mir aktualisiert. Ein Problem existiert noch bei mir unter Windows (ich meinte, dass ich das auch bereits korrigiert hätte, ist offenbar verloren gegangen).

Du hast eine Klasse `edu.andreasgut.sound.AudioPlayer` definiert. Diese Klasse wird im Verzeichnis

    bin/main/edu/andreasgut/sound

abgelegt. Dann hast Du noch ein Verzeichnis mit Sounds unter `resources/edu/andreasgut/Sound`, und diese werden ebenfalls unter `bin` im Verzeichnis

    bin/main/edu/andreasgut/Sound

abgelegt. Unter Windows ist es jedoch nicht möglich zwei Verzeichnisse zu erzeugen mit gleichem Namen aber unterschiedlichem Case, und das führt nun zu einer Fehlermeldung. Ich habe nun den Case der Verzeichnisse unter `src/main/resources` geändert:

    src/main/resources/edu/andreasgut/Images => src/main/resources/edu/andreasgut/images
    src/main/resources/edu/andreasgut/Sound=> src/main/resources/edu/andreasgut/sound

und mit diesen Änderungen kann ich das FX-Programm starten und sehe auch den Button.

Ich habe eben einen entsprechenden Commit gepusht und bitte Dich, den nun in den Master-Branch zu übernehmen.